### PR TITLE
Indentation fix and scripts update

### LINKED ISSUE #23: HomePage.react.js eslint errors
eslint current shows the following errors in `HomePage.react.js` in Atom:

```
Expected indentation of 2 space characters but found 0.
```

for lines 13, 14 and 15. Which doesn't make any sense, they aren't indented differently from the rest. If you change it to two spaces, the errors go away, but 4 are needed for it to look the way it should. No idea why that happens...

### PR DESCRIPTION
**Pull request by yjcxy12**
- fixed some files having both space and tab indent.
- added lint and test:watch to npm scripts

Closes #23

### CODE REVIEW

**Comment by mxstbr** [comment]
Ping to #42, this PR already has the `npm run lint` command 👍
Could you include `npm run lint` in `npm run test`, and add the documentation for `npm run test:watch` and `npm run lint` to `docs/commands.md`?

Thanks so much for contributing, much appreciated!

**Comment by yjcxy12** [comment]
Change the docs as you like. I'm not the best with docs. :)

File: docs/COMMANDS.md [hunk 1]
```diff
@@ -29,7 +29,19 @@ This will run a server that's accessible in the entire local network and shows t
 ## Unit testing
 
 ```Shell
+$ npm run lint
+```
+
+Checks .js files for any style errors.
```
Comment (mxstbr) [inline docs/COMMANDS.md]: Could you change this to `Checks your JavaScript files for potential errors and suspicious usage of included functions` or something to that effect?

I think that's clearer for somebody who doesn't know what linting is and why you should lint your code

File: docs/COMMANDS.md [hunk 2]
```diff
@@ -29,7 +29,19 @@ This will run a server that's accessible in the entire local network and shows t
...
+
+```Shell
 $ npm run test
 ```
 
-Tests your application with the unit tests specified in the `test` folder.
+Lints .js file and tests your application with the unit tests specified in the `test` folder.
```
Comment (mxstbr) [inline docs/COMMANDS.md]: nit: `Lints your JavaScript files`

File: docs/COMMANDS.md [hunk 3]
```diff
@@ -29,7 +29,19 @@ This will run a server that's accessible in the entire local network and shows t
...
+
+```Shell
+$ npm run test:watch
+```
+
+Watches and run tests on .js file changes. To be run on a separate terminal.
```
Comment (mxstbr) [inline docs/COMMANDS.md]: Maybe change it to something to the effect of `Runs unit tests everytime you change something in your JavaScript code`?

**Comment by mxstbr** [comment]
Just a few nitpicks and then we're good to go!

Thanks so much!

**Pull request merged**